src/Psalm/Issue/UnresolvableConstant.php

```diff
@@ -0,0 +1,8 @@
+<?php
+namespace Psalm\Issue;
+
+class UnresolvableConstant extends CodeIssue
+{
+    public const ERROR_LEVEL = -1;
```

Comment:
Should this be set to something else?

Reply:
I believe -1 means that the error will be emitted no matter what level the user is on. It's usually reserved to obvious errors or issue behind a specific flag.

I'm not sure I see yet in what scenario this issue could be emitted, but you have to decide a level based on severity

Reply:
-1 denotes insuppressible issue, -2 - issues controlled by a flag (like `--find-dead-code`).

So -1 should be reserved for certain fatal errors.

Reply:
It should be emitted when `key-of` or `value-of` is used on a constant that can't be resolved, which I think probably should always be emitted, as it's indicative of a typo or other issue with the type annotation, and it's an annotation that's specifically for static analysis. I never use any of the less strict error levels though, so I'm not confident I should be the one deciding.